src/test/java/org/beanpod/switchboard/fixture/ChannelFixture.java

```diff
@@ -55,7 +56,7 @@ public static OutputChannelEntity getOutputChannelEntity() {
     return OutputChannelEntity.builder()
         .id(OUTPUT_CHANNEL_ID)
         .channel(getChannelEntity2())
-        .encoder(EncoderFixture.getEncoderEntity1())
+        .encoder(EncoderFixture.getEncoderEntity2())
```

Comment:
Why the change? Was this necessary for your PR?

Reply:
Yes it's. Previously, both input and output channels fixtures had the same device which makes no sense. Therefore, I changed it so that the output channel has a different device than the input channel.